odoo/api.py

```diff
@@ -751,7 +751,7 @@ def lang(self):
         """
         lang = self.context.get('lang')
         if lang and lang != 'en_US' and not self['res.lang']._get_data(code=lang):
-            raise UserError(_('Invalid language code: %s', lang))
+            raise UserError(self(context={'lang': 'en_US'})._('Invalid language code: %s', lang, _mod=__name__))
```

Comment:
Setting the language to `en_US` prevents translation.  So it should be equivalent to this, shouldn't it?
```suggestion
            raise UserError('Invalid language code: %s' % lang)
```

Reply:
```suggestion
            raise UserError(_('Invalid language code: %s'), lang)
```

Reply:
I will go with a simple f-string and disable the lint warning with explanation why.
Using `_` if you don't have a language will just fail.